FastSimulation/Tracking/plugins/TrackCandidateProducer.cc

```diff
@@ -85,7 +85,8 @@ TrackCandidateProducer::produce(edm::Event& e, const edm::EventSetup& es) {
 
   edm::ESHandle<Propagator>             propagator;
   es.get<TrackingComponentsRecord>().get(propagatorLabel,propagator);
-    
+  Propagator* thePropagator = propagator.product()->clone();
```

Comment:
Matthias is right: this is a memory leak:
another reason to get simply the right propagator from the eventSetup,
as I already suggested earlier.

Thanks!